frontend/plan/src/components/schedule/Stats.js

```diff
@@ -57,22 +51,22 @@ class Stats extends Component {
             if (str) {
                 const course = str.substring(0, str.indexOf("-", str.indexOf("-") + 1)); // finds course (irrespective of section)
                 if (course in courseDifficulties) {
-                    courseDifficulties[course] += (section.difficulty ? section.difficulty : 2.0);
+                    courseDifficulties[course] += (section.difficulty ? section.difficulty : 2.5);
                     courseWorkloads[course] += (section.work_required
-                        ? section.work_required : 2.0);
+                        ? section.work_required : 2.5);
                     courseInstructorQualities[course] += (section.instructor_quality
-                        ? section.instructor_quality : 2.0);
+                        ? section.instructor_quality : 2.5);
                     courseQualities[course] += (section.course_quality
-                        ? section.course_quality : 2.0);
+                        ? section.course_quality : 2.5);
                     courseCUs[course] += (section.credits ? section.credits : 1);
                     courseRepeats[course] += 1;
                 } else {
-                    courseDifficulties[course] = (section.difficulty ? section.difficulty : 2.0);
-                    courseWorkloads[course] = (section.work_required ? section.work_required : 2.0);
+                    courseDifficulties[course] = (section.difficulty ? section.difficulty : 2.5);
```

Comment:
could we generalize a lot of this code? it seems like a lot is copy/paste and for maintainability I think it would be good to abstract these if statements and other logic out into functions, and iterate through with for loops.